templates/primary-secondary/primary-secondary.js

```diff
@@ -1003,16 +1074,20 @@ class TemplatePrimarySecondary extends FocusVisiblePolyfillMixin(RtlMixin(Locali
 			this._desktopKeyboardResizer.secondaryFirst = this.secondaryFirst;
 			this._desktopMouseResizer.secondaryFirst = this.secondaryFirst;
 		}
-		if (!this._secondary) {
-			this._secondary = this.shadowRoot.querySelector('aside');
+		if (!this._divider) {
 			this._divider = this.shadowRoot.querySelector('.d2l-template-primary-secondary-divider');
 		}
+		if (changedProperties.has('_isMobile') && this._isMobile) {
+			this._secondary = this.shadowRoot.querySelector('aside');
+			if (this._divider.isConnected && this._secondary.isConnected) {
+				this._mobileTouchResizer.connect(this._divider, this._secondary);
+			}
+		}
```

Comment:
This solves an issue where going between mobile and desktop mode would create a new secondary panel DOM element so the resizing would no longer work. Now when switching to mobile view, it will re-grab the secondary panel to send to the resizer. Though this was not an issue previously, so I'm not sure what changed.